src/dinners/290117-ood-dinner.js

```diff
@@ -1,9 +1,11 @@
 var Mauro = require('./../attenders/MauroMandracchia'),
-    MontePulcianoWine = require('./../recipes/wine-monte_pulciano'),
-    PastaCarbonara = require('./../recipes/pasta-carbonara'),
-    DessertTiramisu = require('./../recipes/dessert-tiramisu');
+  Eyad = require('./../attenders/EyadKhamis'),
+  MontePulcianoWine = require('./../recipes/wine-monte_pulciano'),
+  PastaCarbonara = require('./../recipes/pasta-carbonara'),
+  DessertTiramisu = require('./../recipes/dessert-tiramisu');
 
-Mauro.brings([ MontePulcianoWine, PastaCarbonara, DessertTiramisu ]);
+Mauro.brings([MontePulcianoWine, PastaCarbonara, DessertTiramisu]);
+Eyad.brings([SpaRood, SpaBlue]);
```

Comment:
Eyad, nice attempt, but... no. SpaRood and SpaBlue are undefined. So try to bring something that at least is defined.